build/toolchain-aarch64-linux-android.cmake

```diff
@@ -31,7 +31,7 @@ set(DIST_ARCH arm64-v8a)
 string(TOLOWER ${CMAKE_HOST_SYSTEM_NAME} HOST_NAME_L)
 file(TO_CMAKE_PATH $ENV{ANDROID_HOME} ANDROID_HOME_UNIX)
 
-file(GLOB NDK_VERSIONS LIST_DIRECTORIES true ${ANDROID_HOME_UNIX}/ndk/*)
+file(GLOB NDK_VERSIONS LIST_DIRECTORIES true ${ANDROID_HOME_UNIX}/ndk/$ENV{FILAMENT_NDK_VERSION}*)
```

Comment:
So this requires setting and exporting FILAMENT_NDK_VERSION. I'd rather build.sh sets is explicitly when invoking CMake with a -D or better yet, why not read the ndk.version file here? That would keep the CMake files "standalone".

Reply:
Reading `ndk.version` from here technically can be done, if `FILAMENT_NDK_VERSION` is not set, as a final fallback. Yet that way `ndk.version` would be read in 3 places. And since this code is in 4 files, for sure some refactoring would be needed to make that nice and tidy.

Reply:
In any case, what next steps you have in mind?

Reply:
I think for now just reading ndk.version would be best. Those 4 files contain a lot of duplicated code, another thing I've been meaning to fix for a long time. But it's not urgent as we rarely touch those files

Reply:
Okay, moved things around and did mini-cleanup

Reply:
Now it would work like that:

```
filament % FILAMENT_NDK_VERSION=28 ./build.sh -i -p android release
Error: Android NDK side-by-side version 28 or higher must be installed, exiting
```

but

```
filament % FILAMENT_NDK_VERSION=22 ./build.sh -i -p android release
```
and
```
filament % ./build.sh -i -p android release
```

would succeed if that NDK 22 is present.

Also, this would work for checking with minor NDK version changes

```
filament % FILAMENT_NDK_VERSION=22.2 ./build.sh -i -p android release
Error: Android NDK side-by-side version 22.2 or higher must be installed, exiting
```